baseapp/baseapp.go

```diff
@@ -681,18 +689,12 @@ func (app *BaseApp) runTx(mode runTxMode, txBytes []byte) (gInfo sdk.GasInfo, re
 			ctx = newCtx.WithMultiStore(ms)
 		}
 
-		events := ctx.EventManager().Events()
```

Comment:
Can you explain the rationale behind these changes? We get the gas and update the context on purpose prior to returning on error.

Reply:
Also, your changes break the comment in the `if !newCtx.IsZero() {` clause.

Reply:
@Vvaradinov this seems to be unintended deletion. Could you undo this line change?